docs/predicate/slsa/v2.md

```diff
@@ -0,0 +1,621 @@
+<!--
+---
+linkTitle: "Chains:v2"
+weight: 10
+---
+-->
+
+# Build Type
+
+This buildType is maintained by [TektonCD/Chains](https://github.com/tektoncd/chains) and describes a build executed by [TektonCD/Pipelines](https://github.com/tektoncd/pipeline).
+
+
+# Description
+
+```
+"buildType": "https://tekton.dev/predicate/chains/slsa/v2"
+```
+
+This buildType describes the execution of a build via a `PipelineRun` or a `TaskRun`.
+
+# Build Definition
+
+## External Parameters
+
+All external parameters are REQUIRED unless otherwise noted. These are fields set by the users in their `PipelineRun/TaskRun` definition.
+
+
+Parameter | Type | Description
+-----------|------|------------
+runSpec | object | complete [pipelineRunSpec](https://tekton.dev/docs/pipelines/pipelineruns/)/[taskRunSpec](https://tekton.dev/docs/pipelines/taskruns/). 
+[buildConfigSource](#build_config_source) | object | The resolved reference to the external build config source.
+
+### Build Config Source
+Parameter | Type | Description
+-----------|------|------------
+ref | string | <li>git reference (i.e. commit sha - <code>"sha1:xxxx"</code>)</li><li>image digest (e.g. <code>"sha256:91fb5e20325059e8feb48c966a83d616596054c5edf811b5bc673683e6ecabb6"</code>)</li>
+repository | string | git repository uri or image repository uri
+path | string | path in the git repo or the resource name in the image bundle.
+
+## Internal Parameters
+
+These are fields provided by the builder for the build process. 
+
+Parameter | Type | Description
+-----------|------|------------
+[FeatureFlags](https://tekton.dev/docs/pipelines/additional-configs/#customizing-the-pipelines-controller-behavior)| object | Tekton configuration feature flags used for the build. These are configured by the service providers and are useful for reproducibility.
+
+## Resolved Dependencies
+
+The resolvedDependencies contains artifacts known to be input to the build. And the name will be used to indicate the type of the artifact. The name of the resolved dependencies is used to distinguish between the different input types to the build as shown below:
+
+Name | Description
+-----|-------------
+**pipeline**|Resolved reference to the remote pipeline.
```

Comment:
Perhaps worth explaining a bit what resolved reference means here? i.e. it has immutable reference of where the remote pipeline came from?